### ISSUE TITLE
<<< PyData Global 2020 Sprint >>>

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#1437]
<img src="https://global.pydata.org/assets/images/pydata.png" width="30"/> <img src="https://raw.githubusercontent.com/pytorch/ignite/master/assets/logo/ignite_logomark.png" width="30"/> We are pleased to announce that we will run a mentored sprint session to contribute to PyTorch-Ignite at [PyData Global 2020](https://global.pydata.org). We are looking forward to seeing you on November 15, 1pm-5pm (UTC+1) !

In this mentored sprint session we aim to introduce the project and help you to start contributing.

Keywords: AI, Machine Learning, Deep Learning, Python, PyTorch

### Learn about PyTorch-Ignite project

PyTorch-Ignite is a high-level library to help with training and evaluating neural networks in PyTorch flexibly and transparently.
Please, read and/or execute in Colab the first two parts ("PyTorch-Ignite: What and Why?",  "Quick-start example") of this [blog post](https://labs.quansight.org/blog/2020/09/pytorch-ignite/) to understand better the project. Feel free to reach out for any questions (see below for details).

To get inspired of other projects who uses this library, please see:
- https://github.com/pytorch/ignite#projects-using-ignite
- https://github.com/pytorch/ignite/network/dependents?package_id=UGFja2FnZS02NzI5ODEwNA%3D%3D

Once, you have an idea of what does this project and you would like to help us to move the project forward, please go to the next section.

### Setup development environment

Please, follow these guidelines: 
- https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#quickstart-guide-for-first-time-contributors
- https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#installation

to install miniconda, setup a conda environment and install necessary development dependencies.

### PyDataGlobal issues

We labeled several GitHub issues with `PyDataGlobal` label for this sprint. They are not of the same difficulty and entry level, so, please read the content inside the issue. They can be about reviewing and improving docs, working on github actions, coding in python, making mypy happy etc. 

Non-tech but very valuable issues (each can be assigned to several persons):
- https://github.com/pytorch/ignite/issues/1438
- https://github.com/pytorch/ignite/issues/1439

Conda Packaging issue:
- https://github.com/pytorch/ignite/issues/1443

List of all PyDataGlobal issues:
-  https://github.com/pytorch/ignite/issues?q=is%3Aissue+is%3Aopen+label%3APyDataGlobal

### Communications during the sprint

If you'd like to reach out on PyData Global mattermost, please send messages to "[SPRINT] PyTorch-Ignite" channel or DM to "vfdev-5".

## PyTorch-Ignite contributing guide:

- https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md

## Other information

Event:
- https://global.pydata.org/talks/270

Mentored Sprints Community Handbook:
- https://mentored-sprints.netlify.app/

The project is currently maintained by a team of volunteers and we are looking for motivated contributors to help us to move the project forward. Please see [About us](https://pytorch.org/ignite/master/about.html) for more information if this sounds interesting to you.
